mParticle-OneTrust/MPKitOneTrust.m

```diff
@@ -45,6 +46,23 @@ - (void)start {
         
         self->_started = YES;
         
+        // READ consent data mapping
+        NSString *mpConsentMapping = [[NSUserDefaults standardUserDefaults] stringForKey:@"OT_mP_Mapping"];
```

Comment:
Thought it would be worth mentioning that I typically favor declaring keys like this one as a constant (or define) in a common area to allow for ease of discovery, reuse, and consistency.  

That said, it looks like we are declaring keys like this one inline in several places already. From a consistency standpoint, it makes sense to leave this alone in that context.

Reply:
I agree with your approach but I personally think stuff like this should probably be done in a separate clean up PR. I think there's other clean up we should do as well, and that can all be lumped together.

Reply:
Agreed!